pwa/src/components/applications/applicationForm.tsx

```diff
@@ -30,6 +30,7 @@ export default function ApplicationForm({ id }) {
   const [alert, setAlert] = React.useState<Record<string, string>>(null);
   const [application, setApplication] = React.useState<IApplication>(null);
   const [showSpinner, setShowSpinner] = React.useState<boolean>(false);
+  let title;
```

Comment:
Deze kan je omschrijven naar:

`const title:string = (id === "new") ? "Create Application" : "Edit Application"`

En dan kan je if else statement op regels 125-129 helemaal weg.

**Note:** we geven hier nu dus ook aan dat `title` als type `string` heeft, hierdoor snapt Typescript dat dit een string is (en wanneer iets of iemand hier e.g. een `number` in probeert te vullen geeft onze editor al direct een error).